sympy/core/numbers.py

```diff
@@ -2447,6 +2447,11 @@ class AlgebraicNumber(Expr):
     is_algebraic = True
     is_number = True
 
+    # Optional alias symbol is not free.
+    # Actually, alias should be a Str, but some methods
+    # expect that it be an instance of Expr.
+    free_symbols = set()
```

Comment:
Which methods are those? Maybe they should be changed.

Reply:
There were some errors when I tried to use Str (I forget, what exactly) but I suppose that it would not be hard to fix them.